web/beacon-app/src/components/MenuDropdown/MenuDropdown.tsx

```diff
@@ -37,40 +37,38 @@ const MenuDropdownMenu = ({ items, isOpen, onOpenChange }: RadixDropdownMenuProp
                 onClick={onClick}
                 className={clsx(
                   'flex cursor-default select-none items-center rounded-md px-2 py-2 text-xs outline-none',
-                  'focus:bg-gray-50 text-gray-400 dark:text-gray-500 dark:focus:bg-gray-900'
+                  'text-gray-500 focus:bg-gray-900'
                 )}
               >
                 {icon}
-                <span className="flex-grow text-gray-700 dark:text-gray-300">{label}</span>
+                <span className="flex-grow text-gray-300">{label}</span>
                 {shortcut && <span className="text-xs">{shortcut}</span>}
               </DropdownMenuPrimitive.Item>
             ))}
 
             {items?.organizationMenuItems?.length > 0 && (
               <>
-                <DropdownMenuPrimitive.Separator className="my-1 h-px bg-gray-200 dark:bg-gray-700" />
-                <DropdownMenuPrimitive.Label className="select-none px-2 py-2 text-xs text-gray-700 dark:text-gray-200">
+                <DropdownMenuPrimitive.Separator className="my-1 h-px bg-gray-700" />
```

Comment:
```suggestion
                <DropdownMenuPrimitive.Separator className={
                mergeClassnames(
                'my-1 h-px'.
                isDarkTheme ? 'bg-gray-700' : 'bg-gray-200'
                )
        
                }
                />
```

Reply:
Following the discussion during today's meeting, I've only changed the hover color to make it more visible and reversed the other changes that were made.